server.go

```diff
@@ -2722,6 +2734,13 @@ func (s *server) addPeer(p *peer) {
 	} else {
 		s.outboundPeers[pubStr] = p
 	}
+
+	// Inform the peer notifier of a peer online event so that it can be reported
+	// to clients listening for peer events.
+	var pubKey [33]byte
+	copy(pubKey[:], p.addr.IdentityKey.SerializeCompressed())
```

Comment:
optimization: do this before creating the `pubStr` above, so don't we have to serialize twice :)